x-pack/plugins/apm/server/routes/metrics/by_agent/serverless/cold_start_duration.ts

```diff
@@ -64,4 +65,24 @@ export function getColdStartDuration({
     additionalFilters: [{ exists: { field: FAAS_COLDSTART_DURATION } }],
     operationName: 'get_cold_start_duration',
   });
+
+  const [series] = coldStartDurationMetric.series;
+
+  const data = series.data.map(({ x, y }) => ({
+    x,
+    // Cold start duration duration is stored in ms, convert it to microseconds so it uses the same unit as the other charts
+    y: isFiniteNumber(y) ? y * 1000 : y,
+  }));
+
+  return {
+    ...coldStartDurationMetric,
+    series: [
+      {
+        ...series,
+        // Cold start duration duration is stored in ms, convert it to microseconds
+        overallValue: series.overallValue * 1000,
+        data,
+      },
+    ],
```

Comment:
Maybe we can align with that for `8.6`?  They are going to refactor the way the aws lambda is stored anyways.

cc: @AlexanderWert Do you think it's going to be possible to store the duration fields in microseconds?

Reply:
I think this can be changed easily on the extension, though the metrics would not align with the unit they are reported in CloudWatch. Not sure if that would be confusing for users if the would use same metrics for example in custom dashboards

Reply:
Got it. Let's leave it like this for now and have a discussion later about it.